openssl-dynamic/src/main/java/io/netty/internal/tcnative/Library.java

```diff
@@ -119,16 +107,7 @@ public static boolean initialize() throws Exception {
     public static boolean initialize(String libraryName, String engine) throws Exception {
         if (_instance == null) {
             _instance = libraryName == null ? new Library() : new Library(libraryName);
-
-            if (aprMajorVersion() < 1) {
-                throw new UnsatisfiedLinkError("Unsupported APR Version (" +
-                                               aprVersionString() + ")");
-            }
-
-            if (!aprHasThreads()) {
-                throw new UnsatisfiedLinkError("Missing APR_HAS_THREADS");
-            }
         }
-        return initialize0() && SSL.initialize(engine) == 0;
+        return SSL.initialize(engine) == 0;
```

Comment:
consider just changing `SSL.initialize` to a boolean which returns true if it has already been initialized or this call successfully initialized the `SSL` module.